cypress/e2e/demo/demoDashboard.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress"/>
+/// <reference types="../../support"/>
+
+import {
+  BUTTON_SELECTORS,
+  HOMEPAGE_SELECTORS,
+  SHARED_ELEMENTS,
+} from "../../elements";
+import { orderDraftCreateDemoResponse, urlList } from "../../fixtures";
+
+describe("Dashboard demo site tests", () => {
+  it("should be able to log in via UI", () => {
+    //   { tags: ["@login", "@allEnv", "@stable", "@oldRelease"] },
+
+    cy.addAliasToGraphRequest("Home")
+      .visit(urlList.dashboard)
+      .get(BUTTON_SELECTORS.submit)
+      .should("be.visible")
+      .click()
+      .waitForRequestAndCheckIfNoErrors("@Home");
+    cy.get(SHARED_ELEMENTS.notificationMessage).should("have.length", 1);
+    cy.get(HOMEPAGE_SELECTORS.welcomeMessage).should("be.visible");
+    cy.get(SHARED_ELEMENTS.notificationMessage).should("not.exist");
+  });
+  it("should not be able to create new order", () => {
+    //   { tags: ["@login", "@allEnv", "@stable", "@oldRelease"] },
+
+    cy.addAliasToGraphRequest("OrderList");
+    cy.loginUserViaRequest().then(() => {
+      cy.visit(`${urlList.dashboard}/orders`).waitForRequestAndCheckIfNoErrors(
+        "@OrderList",
+      );
+      cy.pickAndSelectChannelOnCreateOrderFormByIndex(1);
```

Comment:
Why it's a command? From cypress docs: "Custom commands work well when you're needing to describe behavior that's desirable across **all of your tests**. Custom commands you write are generally an abstraction over a series of internal commands. That means you and your team members exert much more mental effort to understand what your custom command does. Try not to overcomplicate things and create too many abstractions. When in doubt, use a regular function for individual spec files."